doc/lsst.pipe.tasks/tasks/lsst.pipe.tasks.postprocess.ConsolidateSourceTableTask.rst

```diff
@@ -0,0 +1,102 @@
+.. lsst-task-topic:: lsst.pipe.tasks.postprocess.ConsolidateSourceTableTask
+
+##########################
+ConsolidateSourceTableTask
+##########################
+
+``ConsolidateSourceTableTask`` concatenates per-detector Source Tables (dataset `sourceTable`) into one per-visit Source Table (dataset `sourceTable_visit`).
+It only does I/O, and therefore has no run method.
+The inputs have already been transformed to the DPDD-specified columns, and this task assumes that they are sufficiently narrow to fit all tables for a given visit in memory at once.
+
```

Comment:
You should also mention here that this is the third step, and put links in to the first two steps.